_includes/azerothcore/sidebar.html

```diff
@@ -68,6 +68,7 @@
                 <li><a href="/wiki/remote-access">Remote Access</a></li>
                 <li><a href="/wiki/waypoints-information">Information about waypoints</a></li>
                 <li><a href="/wiki/how-to-obtain-free-records">How to obtain free records</a></li>
+                <li><a href="/wiki/spell-effects-reference">Spell Effects Reference</a></li>
```

Comment:
Trivial but my opinion is of limiting the use of the sidebar and depending more on the documentation index page.

Reply:
I personally didn't even know the documentation page existed so I'd rather leave it in both places, can always be removed.